common/src/androidTest/java/com/microsoft/identity/common/CommandDispatcherTest.java

```diff
@@ -365,6 +369,48 @@ public void run() {
         }
     }
 
+    @Test
+    public void testCanRecoverFromHangingTask() throws Exception {
+        final AtomicInteger integer = new AtomicInteger(0);
```

Comment:
Can we use a different name than `integer` as that may at first seem like the data type rather than the variable name